pages/whom-we-are-hiring.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Who are we hiring?
```

Comment:
If you're going to change the URL, you should change the title.
If you change the title, it becomes "Whom are we hiring?" and we become an elderly dowager looking down through our moon-rimmed glasses at the impertinent urchins who presume to earn our favour.

(For the final word, I hereby invoke @kategarklavs)